### PR TITLE
move ci from travis to gitaction

### PR DESCRIPTION
**Pull request by wy65701436**
1, deprecate travis, and use the gitaction for instread.
2, upgrade golang version to v1.15.6

Signed-off-by: Wang Yan <wangyan@vmware.com>

### CODE REVIEW

**Review by bitsf — APPROVED** [review]

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+  APITEST_DB:
+    env:
+      APITEST_DB: true
+    runs-on:
+      #- self-hosted
+      - ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Set up Go 1.14
```
Comment (reasonerjt) [inline .github/workflows/CI.yml:98]: 1.15

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+  APITEST_LDAP:
+    env:
+      APITEST_LDAP: true
+    runs-on:
+      #- self-hosted
+      - ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Set up Go 1.14
```
Comment (reasonerjt) [inline .github/workflows/CI.yml:158]: 1.15

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+  OFFLINE:
+    env:
+      OFFLINE: true
+    runs-on:
+      #- self-hosted
+      - ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Set up Go 1.14
```
Comment (reasonerjt) [inline .github/workflows/CI.yml:216]: 1.15

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -127,7 +127,7 @@ Harbor backend is written in [Go](http://golang.org/). If you don't have a Harbo
 |   1.6    |    1.9.2      |
 |   1.7    |    1.9.2      |
 |   1.8    |    1.11.2     |
-|   1.9    |    1.12.12    |
+|   1.9    |    1.15.6    |
```
Comment (reasonerjt) [inline CONTRIBUTING.md:130]: we didn't bump up 1.9

**Review by reasonerjt — APPROVED** [review]

**Pull request merged**